documentation/release-notes/topics/version_4.rst

```diff
@@ -70,6 +70,11 @@ Other notable changes and improvements
 
 - We added a shortcut to the asset sensors list from the Information menu: *Information -> Sensors List*.
 
+============
+Known issues
+============
+- **AS-14401** Custom property columns will appear in the order they are added to the grid and not in alphabetical order. 
```

Comment:
Actually this is not exactly true either.

It is this and a combination of prior state in saved grid.  

May be best to say 

Custom property columns may not appear in alphabetical order.

Reply:
Thanks, updated verbiage.